pytest_terraform/__init__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2020 Kapil Thangavelu
```

Comment:
none of the code in this package should be in this pr. its a separate repository/dependency.